cli/src/semgrep/meta.py

```diff
@@ -203,6 +208,7 @@ def to_project_metadata(self) -> out.ProjectMetadata:
             # REQUIRED for semgrep-app backend
             repository=self.repo_name,
             # OPTIONAL for semgrep-app backend
+            repo_display_name=self.repo_display_name,
```

Comment:
nit: I think this should move up a line and be considered "required" if we are going to have it fall back on self.repo_name. Alternatively, you could have it not do that, and be of type `str | None` and leave it here.